### PR TITLE
Add Low Altitude Disaster Imagery (LADI) dataset to dataset registry

### PR DESCRIPTION
**Pull request by aweinert-MIT**
Aligns with commit ladi-overview commit 044bba3dceccafe7d1354573fdd1e9ff9e0a4749

*Issue #, if available:*
None

*Description of changes:*
Create .yaml for the Low Altitude Disaster Imagery (LADI) dataset

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: e80ba814fedc96ec900fbbe8031f5fe18ae80358
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=117bdf7c-7009-4d23-b9bc-88462a763254%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: datasets/ladi.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+Name: Low Altitude Disaster Imagery (LADI) Dataset
+Description: The Low Altitude Disaster Imagery (LADI) Dataset consists of human and machine annotated airborne images collected by the Civil Air Patrol in support of various disaster responses from 2015-2019. The initial release of LADI focuses on the Atlantic hurricane seasons and coastal states along the Atlantic Ocean and Gulf of Mexico. Annotations are included for major hurricanes of Harvey, Maria, and Florence. Two key distinctions are the low altitude, oblique perspective of the imagery and disaster-related features, which are rarely featured in computer vision benchmarks and datasets.
+Documentation: https://github.com/LADI-Dataset/ladi-overview/README.md
```
Comment (jflasher) [inline datasets/ladi.yaml:3]: Looks like this link is not resolving.

File: datasets/ladi.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+Contact:  ladi-dataset-admin@mit.edu
+ManagedBy: "[MIT Beaver Works](https://beaverworks.ll.mit.edu/)"
+UpdateFrequency: Periodically
+License: Creative Commons 4.0 (CY4.0)
```
Comment (jflasher) [inline datasets/ladi.yaml:7]: Double presence of license field, I think you can remove this one.

File: datasets/ladi.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+      URL: https://www-nlpir.nist.gov/projects/tv2020/dsdi.html
+      AuthorName: TREC Video Retrieval Evaluation (TRECVID) 
+  Publications:
+   - Title: Large Scale Organization and Inference of an Imagery Dataset for Public Safety
+      URL: https://doi.org/10.1109/HPEC.2019.8916437
+      AuthorName: Jeffrey Liu, David Strohschein, Siddharth Samsi, Andrew Weinert
+    - Title: Train and Deploy an Image Classifier for Disaster Response
+      URL: https://arxiv.org/abs/2005.05495
+      AuthorName: Jianyu Mao, Kiana Harris, Nae-Rong Chang, Caleb Pennell, Yiming Ren
+    - Title: Integrating Multiple Deep Learning Models to Classify Disaster Scene Videos
```
Comment (jflasher) [inline datasets/ladi.yaml:51]: URL will be required for both of these DataAtWork entries.

**Review by jflasher — CHANGES_REQUESTED** [review]
Hi @aweinert-MIT, just a few comments and then I think it should be good to merge, thanks!

File: datasets/ladi.yaml (hunk 3 above)
Comment (aweinert-MIT) [reply datasets/ladi.yaml:51]: These were presented at the IEEE HPEC conference next week but they're not available via IEEE xplore. Should I remove them here and update once the papers are available or should I use the conference website for the URL?

Comment (jflasher) [reply datasets/ladi.yaml:51]: Yeah, I think best to just remove for now, thanks!

Comment (aweinert-MIT) [reply datasets/ladi.yaml]: Removed in commit 561feea

File: datasets/ladi.yaml (hunk 2 above)
Comment (aweinert-MIT) [reply datasets/ladi.yaml]: Removed in commit 561feea

File: datasets/ladi.yaml (hunk 1 above)
Comment (aweinert-MIT) [reply datasets/ladi.yaml]: Addressed by commit 561feea

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 561feeac603e205e2ba696b43b9fe48e93e4538f
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=b421a285-60c4-4f30-8102-263c9bce2825%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: datasets/ladi.yaml [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  - land
+  - machine learning
+  - mapping
+  - natural resource
+  - seismology
+  - transportation
+  - urban
+  - water
+Resources:
+  - Description: LADI dataset: images and labels
```
Comment (zflamig) [inline datasets/ladi.yaml:29]: We need to add quotes around "LADI dataset: images and labels" here since there is a : there that confuses the yaml parser

Comment (aweinert-MIT) [reply datasets/ladi.yaml]: Addressed in commit 33e04f0

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 33e04f0b605e28f0b802948539374f70cd734add
* Result: FAILED
* [Build Logs](https://ksf1fa59z7.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=8ff06cd0-f98b-4c92-916c-2e3ff5c43dba%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: datasets/ladi.yaml [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+    - Title: NIST TRECVID 2020 - Disaster Scene Description and Indexing (DSDI)
+      URL: https://www-nlpir.nist.gov/projects/tv2020/dsdi.html
+      AuthorName: TREC Video Retrieval Evaluation (TRECVID)
+    - Title: Remote Sensing for Disaster Response Course
+      URL: https://github.com/bwsi-hadr
+      AuthorName: Beaver Works Summer Institute
+      AuthorURL: https://beaverworks.ll.mit.edu/CMS/bw/bwsi
+  Publications:
+   - Title: Large Scale Organization and Inference of an Imagery Dataset for Public Safety
+      URL: https://doi.org/10.1109/HPEC.2019.8916437
```
Comment (zflamig) [inline datasets/ladi.yaml:49]: Indentation is off here, need to remove a space

File: datasets/ladi.yaml [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+      AuthorName: Jeffrey Liu, David Strohschein, Siddharth Samsi, Andrew Weinert
```
Comment (zflamig) [inline datasets/ladi.yaml:50]: Indentation is off here, need to remove a space

File: datasets/ladi.yaml [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+   - Title: Video Testing at the FirstNet Innovation and Test Lab Using a Public Safety Dataset
+      URL: https://doi.org/10.1109/LNET.2019.2955833
```
Comment (zflamig) [inline datasets/ladi.yaml:52]: Indentation is off here, need to remove a space

File: datasets/ladi.yaml [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+      AuthorName: Chris Budny, Jeffrey Liu, Andrew Weinert
```
Comment (zflamig) [inline datasets/ladi.yaml:53]: Indentation is off here, need to remove a space

File: datasets/ladi.yaml [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+    - Title: Train and Deploy an Image Classifier for Disaster Response
```
Comment (zflamig) [inline datasets/ladi.yaml:54]: These ones aren't aligned with above too

Comment (aweinert-MIT) [reply datasets/ladi.yaml]: Addressed in fbe186b

File: datasets/ladi.yaml (hunk 8 above)
Comment (aweinert-MIT) [reply datasets/ladi.yaml]: Addressed in fbe186b

File: datasets/ladi.yaml (hunk 7 above)
Comment (aweinert-MIT) [reply datasets/ladi.yaml]: Addressed in fbe186b

File: datasets/ladi.yaml (hunk 6 above)
Comment (aweinert-MIT) [reply datasets/ladi.yaml]: Addressed in fbe186b

File: datasets/ladi.yaml (hunk 5 above)
Comment (aweinert-MIT) [reply datasets/ladi.yaml]: Addressed in fbe186b

**Pull request merged**